_data/site_menu.yml

```
@@ -31,11 +31,14 @@ installation:
       path: "/docs/reference/latest/installing_on_scvmm/index.html"
       desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft System Center Virtual Machine Manager (SCVMM)"
 
-upgrading:
-  title: Upgrading
+appliances:
+  title: Appliances
   children:
-    - title: Appliances
-      path: "/docs/reference/latest/upgrading_appliances/index.html"
+    - title: Backup and Restore
+      path: "/docs/reference/latest/appliances/backup_and_restore.html"
+      desc: "How to backup and restore appliances"
+    - title: Upgrading
+      path: "/docs/reference/latest/appliances/upgrading.html"
```

Since this path was moved from a previous location, also add a "prior" here:

```suggestion
      path: "/docs/reference/latest/appliances/upgrading.html"
      prior: "/docs/reference/quinteros/upgrading_appliances/index.html"
```

if we plan to backport to quinteros or petrosian, then we'll have to change which one is "prior".  You only need one "prior" in the historical chain, so it would only be in the version we choose to first rename.

---

This is needed because when we generate the site, we determine which site is the canonical version, and we list all other versions as being "old" versions of the newest.  This helps with Google SEO, so it doesn't index the "old" stuff and gives preference to the "new" stuff.

---

Done